examples/sunlinsol/ginkgo/test_sunlinsol_ginkgo.cpp

```diff
@@ -143,10 +143,10 @@ __global__ void fill_kernel(sunindextype mat_rows, sunindextype mat_cols,
 }
 #endif
 
-void fill_matrix(gko::matrix::Csr<sunrealtype, sunindextype>* matrix)
+static void fill_matrix(std::shared_ptr<gko::matrix::Csr<sunrealtype, sunindextype>>& matrix)
```

Comment:
@balos1 Not sure if this is the right fix but I couldn't compile this test with Ginkgo 1.7 and GCC 13.2 without this update.

Reply:
Is the reference needed?

Reply:
Nope